client/components/web-preview/content.jsx

```diff
@@ -394,6 +394,7 @@ export default class WebPreviewContent extends Component {
 				{ this.props.showExternal && this.props.isModalWindow && ! this.props.isPrivateAtomic && (
 					<DomainUpsellCallout trackEvent="site_preview_domain_upsell_callout" />
```

Comment:
I haven't followed all the conversation about adding this feature/upsell here. But I am a bit surprised to see a DomainUpsellCallout component just above where we're adding another domain upsell. Would it have been possible to use this existing domain upsell component?